metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/profiler_master.py

```diff
@@ -59,6 +60,9 @@ def configure(self, env, upgrade_type=None, config_dir=None):
             commands.create_hbase_tables()
         if params.security_enabled and not commands.is_hbase_acl_configured():
             commands.set_hbase_acls()
```

Comment:
Is there a reason that we do not call `commands.set_hbase_acl_configured()` here?

Reply:
Sorry to bring this up on an old, closed PR, but I am currently trying to track a bug that is giving me an uber headache. :)  Thought this was the easiest way to reach out.

Reply:
@nickwallen It happens at the end of `set_hbase_acls()`

Reply:
Ah, daft, you're right.  It works different than the Kafka ACL functions.